src/clients/Fabric.js

```diff
@@ -1088,6 +1088,35 @@ const Fabric = {
 
   /* Object creation / modification */
 
+  GetContentAdminsGroupAddress: async () => {
+    try {
+      // Attempt to add newly created object to content admins group
+      const tenantContractId = await client.userProfileClient.TenantContractId();
+      const contentAdminGroupAddress = await client.CallContractMethod({
+        contractAddress: client.utils.HashToAddress(tenantContractId),
+        methodName: "groupsMapping",
+        methodArgs: ["content_admin", 0],
+        formatArguments: true,
+      });
+
+      if(!contentAdminGroupAddress) {
+        throw "Unable to determine content admins group address";
+      }
+
+      const userGroups = await client.Collection({collectionType: "accessGroups"});
```

Comment:
I don't think we need to worry about the user being member of the content admins group.
A user that is not a content admin should be able to make a new object and give permissions to the content admins.

The one thing we need to do in the future is stronger validation that this user is part of the tenant.  But right now we don't have the necessary pieces to do it just yet.

Reply:
you don't need to be a member of the group to grant permissions? ok